pywbem_mock/_inmemoryrepository.py

```diff
@@ -0,0 +1,279 @@
+#
+# (C) Copyright 2020 InovaDevelopment.comn
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+# Author: Karl  Schopmeyer <inovadevelopment.com>
+#
+
+"""
+The pywbem_mock module provides a standard interface to a CIM repository that
+stores collections of :class:`~pywbem.CIMClass`, :class:`~pywbem.CIMInstance`,
+and :class:`~pywbem.CIMQualifierDeclaration` as Python asbstract classes in the
+module :py:mod:`pywbem_mock/_baserepository.py`.
```

Comment:
1. Typo "asbstract"
2. I don't think we should mention the module name. It should be sufficient to mention that there are abstract base classes. I.e. remove "in the module :py:mod:'pywbem_mock/_baserepository.py'"

Reply:
Removed it in a couple of places so it just referers to abstract classes.

Reply:
DONE